consent/strategy_logout_test.go

```diff
@@ -490,4 +509,24 @@ func TestLogoutFlows(t *testing.T) {
 
 		wg.Wait()
 	})
+
+	t.Run("case=should execute backchannel logout in headless flow with sid", func(t *testing.T) {
+		sid := make(chan string, 2)
```

Comment:
Using a buffered channel here to make sure that we have both values in it and do not block the goroutine that tries to read it.